File: examples/function/delete/go/deleteAFunction.go
package example

import (
	"fmt"
	"github.com/micro/services/clients/go/function"
	"os"
)

// Delete a function by name
func DeleteAfunction() {
	functionService := function.NewFunctionService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := functionService.Delete(&function.DeleteRequest{
		Name:    "my-first-func",
		Project: "tests",
	})
	fmt.Println(rsp, err)
}

File: examples/function/delete/node/deleteAFunction.js
const { FunctionService } = require("m3o/function");

// Delete a function by name
async function deleteAfunction() {
  let functionService = new FunctionService(process.env.MICRO_API_TOKEN);
  let rsp = await functionService.delete({
    name: "my-first-func",
    project: "tests",
  });
  console.log(rsp);
}

deleteAfunction();
